src/registrar/admin.py

```diff
@@ -1852,7 +1876,7 @@ def status_history(self, obj):
 
     # Read only that we'll leverage for CISA Analysts
     analyst_readonly_fields = [
-        "federal_agency",
+        "converted_federal_agency",
```

Comment:
From my understanding, we are stripping out all the ACs involving change forms.  Revert to federal_agency.